### DISCUSSION TITLE
my user happiness story

### DISCUSSION THREAD

**Discussion by ofirgeller** [cline/cline#242] [Feature Requests]
I copied a path of a file to include in the prompt, then i wrote:

this file has such and such that needs to be fixed

forgot to actually paste the path, and it figured it out from the editor state (i think?)
asked for the right file straight away :)

**Comment by saoudrizwan** [comment]
Love to see it! Thanks for sharing :^)

**Comment by zenyum** [comment]
I rammaged this post out to thank the author for this (seemingly) fantastic piece of art. It rescued me from the exhausting frustration by Costly Using ReSOuRce, within one minute, with the cheapest gpt model, with one try, within the past days.

**Comment by zenyum** [comment]
I am trying out FlyingTogether and asked it
```a_question_to_assitanse
if this component function should be wrapped up in a memo hook. 
```

FlyingTogether answered "Yes, you can because xxxx (a general two-sentence explantion)  and here is the code". 

On the contrary, Cline provided a comprehensive clear analysis respecting current implementation, benefit and impact, feasibility and recommendation. With the cheapest gpt-4o-mini.
